app-text/master-pdf-editor/master-pdf-editor-5.7.08.ebuild

```
@@ -0,0 +1,51 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop xdg-utils
```

```suggestion
inherit desktop xdg
```
and remove `pkg_postinst()`, `pkg_postrm()`

---

Thanks for the suggestions, I have made the proposed changes to the ebuild.